coalib/output/Logging.py

```diff
@@ -33,3 +38,53 @@ def configure_logging():
             }
         }
     })
+
+
+def configure_json_logging():
+    """
+    Configures logging as a JSON objects.
+    Returns a StringIO() that captures the logs as json.
+    """
+    import logging.config
+    import io
+
+    stream = io.StringIO()
+
+    logging.config.dictConfig({
+        'version': 1,
+        'handlers': {
+            'json': {
+                'class': 'logging.StreamHandler',
+                'formatter': 'json',
+                'stream': stream
+            }
+        },
+        'root': {
+            'level': 'DEBUG',
+            'handlers': ['json']
+        },
+        'formatters': {
+            'json': {
+                '()': 'coalib.output.Logging.JSONFormatters',
+            }
+        }
+    })
+    return stream
+
+
+class JSONFormatters(logging.Formatter):
+    """
+    JSON formatter for python logging
+    """
+    @staticmethod
+    def format(record):
+        # Create message dict
+        message = {
+            'timestamp': datetime.utcfromtimestamp(record.created).isoformat(),
+            'message': record.getMessage(),
+            # 'path': record.pathname,
+            'level': record.levelname,
+            # 'logger': record.name,
```

Comment:
I kept it for future reference. Do we need the path of the file in which the error originated? 
Also if we add multiple loggers in the future (we use only one now) we might need the logger key.